test/src/browser-pool/index.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
 
-const BasicPool = require("src/browser-pool/basic-pool");
-const LimitedPool = require("src/browser-pool/limited-pool");
-const PerBrowserLimitedPool = require("src/browser-pool/per-browser-limited-pool");
+const BasicPool = require("src/browser-pool/basic-pool").default;
```

Comment:
I'd switch to named exports where possible, since we already have to change "default" in so many places. But that's optional, up to you.